.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 /phpunit.xml
 .phpunit.result.cache
+/.idea
```

Comment:
We don't have this change - on any of our repositories.

Reply:
I saw the .idea was not added in the gitignore so added it.

if you meant something else then plz explain.